src/zenml/integrations/seldon/steps/seldon_deployer.py

```diff
@@ -192,3 +239,151 @@ def prepare_service_config(model_uri: str) -> SeldonDeploymentConfig:
     )
 
     return service
+
+
+@step(enable_cache=False)
+def seldon_custom_model_deployer_step(
+    deploy_decision: bool,
+    config: SeldonDeployerStepConfig,
+    context: StepContext,
+    model: ModelArtifact,
+) -> SeldonDeploymentService:
+    """Seldon Core custom model deployer pipeline step.
```

Comment:
If I recall correctly, this step is actually very similar to the one of Kserve, might be nice if we could abstract the differences away in some functions on the `ModelDeployer` subclasses, but definitely not necessary in this PR

Reply:
I will mark it as the first thing to do just after release, there is a lot of functions that got repeated and can benefit of some cleaning